packages/common/src/lib/common/logger.ts

```diff
@@ -125,24 +152,24 @@ const logger = function (level, files, noStdout, prefix) {
         files = [files];
     }
 
-    const formatter = info => `${timestamp(info.timestamp)} - ${info.level}: ${info.message}`;
+    const formatter = (info: LogInfo) => `${timestamp(info.timestamp)} - ${info.level}: ${info.message}`;
 
     files = files || [];
 
     // indicator which is used to determine the log dir for developing, where it should be inside the repository
     const isNpm = !__dirname
         .replace(/\\/g, '/')
         .toLowerCase()
-        .includes(tools.appName.toLowerCase() + `.js-controller/packages/`);
+        .includes(`${tools.appName.toLowerCase()}.js-controller/packages/`);
 
     if (typeof level === 'object') {
```

Comment:
Better use `isObject` helper. Even if we limit the type to something non-nullable, seeing `typeof ... === "object"` checks makes me nervous :)

Reply:
yes now that tools is ts I can do that, else he wouldn't get that it is of type object after the check :+1: